appdb.py

```diff
@@ -15,6 +15,16 @@
 mysql=MySQL(app)
 
 @app.route('/', methods=['GET','POST'])
+def home():
+    if request.method=='GET':

```

Comment:
you don't need `POST` support in your home :) It can be only `GET`.

Reply:
please take a look at this comment^ you don't need `POST` in `/`. You can remove it from the `methods` and the code itself (`if request.method == 'POST'...`)